fec.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  appUrl: '/insights/registration',
+  debug: true,
+  useProxy: true,
+  proxyVerbose: true,
+  interceptChromeConfig: false,
+  plugins: [],
+  hotReload: process.env.HOT === 'true',
```

Comment:
The same question as for malware https://github.com/RedHatInsights/malware-detection-frontend/pull/111/files#r1486147794

Reply:
Good question, the doc you sent is out of date, `hotReload` replace `_unstableHotReload`, see:
https://github.com/RedHatInsights/frontend-components/blob/4ef5accbd645b6a416f2e2bd31096b05771056ec/packages/config/src/lib/createConfig.ts#L46

or

https://github.com/RedHatInsights/frontend-components/blob/bbef2f308320b4b879020ac43f46a23d07df0574/packages/config/src/bin/webpack.plugins.ts#L10

I'll try to create PR to fix the docs.